src/memray/commands/attach.py

```diff
@@ -332,11 +340,19 @@ def run(self, args: argparse.Namespace, parser: argparse.ArgumentParser) -> None
             live_port = _get_free_port()
             destination = memray.SocketDestination(server_port=live_port)
 
+        if args.aggregate and not hasattr(args, "output"):
+            parser.error("Can't use aggregated mode without an output file.")
```

Comment:
This `parser.error()` never fires, because `hasattr(args, "output")` is always `True`.

Sometimes the attribute is `None`, but it always exists. If it didn't, we'd get an `AttributeError` on this line above: https://github.com/bloomberg/memray/blob/2751eb4700f09700abebb668299260aef1b3b3d2/src/memray/commands/attach.py#L332

We should add a unit test that exercises this case, too, so that we're not introducing a new, uncovered line of code.